### PR TITLE
Enable H014 and H023 djlint rules

### PR DESCRIPTION
**Pull request by silverwind**
Enable these rules:

- H014 | More than 2 blank lines.
- H023 | Do not use entity references.

There are more potential rules to enable but they are blocked by bugs in the linter:

- https://github.com/Riverside-Healthcare/djLint/issues/711
- https://github.com/Riverside-Healthcare/djLint/issues/712

### CODE REVIEW

**Review by techknowlogick — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

File: templates/repo/editor/edit.tmpl [hunk 1]
```diff
@@ -52,8 +52,6 @@
 			{{template "repo/editor/commit_form" .}}
 		</form>
 	</div>
-
-
```
Comment (delvh) [inline templates/repo/editor/edit.tmpl:56]: Why both?

File: templates/repo/settings/options.tmpl [hunk 2]
```diff
@@ -44,30 +44,25 @@
...
 
 			<div class="divider"></div>
-
 			<form class="ui form" action="{{.Link}}/avatar" method="post" enctype="multipart/form-data">
 				{{.CsrfTokenHtml}}
 				<div class="inline field">
 					<label for="avatar">{{.locale.Tr "settings.choose_new_avatar"}}</label>
 					<input name="avatar" type="file" accept="image/png,image/jpeg,image/gif,image/webp">
 				</div>
-
```
Comment (delvh) [inline templates/repo/settings/options.tmpl:61]: The changes in this file look a bit arbitrary?
What's the reason for them?

File: templates/user/auth/change_passwd_inner.tmpl [hunk 3]
```diff
@@ -11,13 +11,10 @@
 				<label for="password">{{.locale.Tr "password"}}</label>
 				<input id="password" name="password" type="password" value="{{.password}}" autocomplete="new-password" required>
 			</div>
-
-
```
Comment (delvh) [inline templates/user/auth/change_passwd_inner.tmpl:15]: Again, both?

**Comment by delvh** [comment]
Apparently I was 10 seconds too late :grin:

**Comment by silverwind** [comment]
Do we really need to discuss whitespace changes? :D

**Comment by delvh** [comment]
Well, what confused me is simply that the rule says `two subsequent empty lines are forbidden`, and you deleted more than necessary?

**Comment by silverwind** [comment]
Yes, I deleted a few more a I saw fit. Generally I think we should have zero empty lines, could likely make a sed replacement for it if appreciated.

**Comment by delvh** [comment]
Actually, blank lines do have a purpose: They separate groups visually for humans.
That's why I'm somewhat against removing more than necessary.

**Pull request merged**